src/modules/profile/ui/VendorProfileForm.tsx

```
@@ -49,13 +61,111 @@ export function VendorProfileForm() {
+      return "Select categories first";
+    }
+    
+    // If categories are selected but no subcategories are chosen
+    return "Select subcategories";
+  };
+
+  // Helper function to determine if we should use black font for better visibility
+  const shouldUseBlackFont = (fieldType: 'services' | 'categories' | 'subcategories') => {
+    switch (fieldType) {
+      case 'services':
+        const currentServices = form.getValues("services") || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)